src/pj_factors.c

```diff
@@ -72,7 +72,7 @@ int pj_factors(LP lp, PJ *P, double h, struct FACTORS *fac) {
 
         /* convergence */
         if (!(fac->code & IS_ANAL_CONV)) {
-            fac->conv = - atan2(fac->der.y_l, fac->der.x_l);
+            fac->conv = - atan2(fac->der.x_p, fac->der.y_p);
```

Comment:
Thanks, I think I get it now. I guess this implies a change in convention? Are the old an new calculations equivalent for conformal projections? In that case I think it is safe to merge. If not, we should consider adding this as a seperate derivative measure of coordinates.

Did you write this your self? Can we include it in the documentation?